src/EditorFeatures/Core/InlineRename/IEditorInlineRenameService.cs

```diff
@@ -72,9 +73,9 @@ internal enum InlineRenameFileRenameInfo
 
     public InlineRenameReplacement(InlineRenameReplacementKind kind, TextSpan originalSpan, TextSpan newSpan) : this()
     {
-        this.Kind = kind;
-        this.OriginalSpan = originalSpan;
-        this.NewSpan = newSpan;
+        Kind = kind;
+        OriginalSpan = originalSpan;
+        NewSpan = newSpan;
```

Comment:
please keep this.   It helps know if it's an instance assignment versus a static one.